ladders.js

```diff
@@ -97,6 +97,10 @@ class Ladder extends LadderStore {
 			connection.popup(message);
 			return null;
 		}
+		if (user.battlebanned) {
```

Comment:
There's no need to have a `.battlebanned` variable. We don't have a `.banned` or `.roombanned` variable for the same reason. We can just check whenever the user tries to start a battle.

Reply:
Kk I’ll fix it although couldn’t you argue the same for ``.locked`` with talking in chats?

Reply:
... locks are a global punishment, I thought we've established that battle bans shouldn't be?

Reply:
....ok? ``.banned`` is global?? I don’t see your point

Reply:
I would argue being battlebanned and lock are pretty close to the same severity, this being a global or a room punishment shouldn’t matter imo

Reply:
I meant, global bans and global locks are global punishments, and battle bans aren't

Reply:
`.locked` is more complicated, it's a variable mostly because it comes with a symbol and so filters can set it.

Reply:
I know @panpawn but it’s irrelevant as ``.banned`` isn't a variable. It seems to only apply to ones that get called a decent amount, which it appears I was mostly correct.